### PR TITLE
Opgops 1706

### PR DESCRIPTION
**Pull request by tim-mcarthur-moj**

### CODE REVIEW

File: ecs/init.sls [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+net.ipv4.conf.all.route_localnet:
+    sysctl.present:
+      - value: 1
+
+iptables-dnat:
+  iptables.append:
+    - table: nat
+    - rule: "-p tcp -d 169.254.170.2 --dport 80 -j DNAT --to-destination 127.0.0.1:51679"
```
Comment (carinadigital) [inline ecs/init.sls:17]: Trying to understand what this rule does and purpose.
IP is AWS metadata service, and we send that to local?

**Review by brettminnie — APPROVED** [review]

File: ecs/init.sls (hunk 1 above)
Comment (carinadigital) [reply ecs/init.sls:17]: I think some comment explanation would be good here

Comment (tim-mcarthur-moj) [reply ecs/init.sls:17]: this is as per the aws docs.  i will put a link into the readme

Comment (tim-mcarthur-moj) [reply ecs/init.sls:17]: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-install.html

**Review by brettminnie — APPROVED** [review]

**Pull request merged**